SOURCES/etc/xapi.d/plugins/xcpngutils/__init__.py

```diff
@@ -106,7 +124,7 @@ def wrapper(*args, **kwds):
             message = e.strerror if e.strerror is not None else str(e.args)
             raise XenAPIPlugin.Failure(str(e.errno), [message, str(e.filename), traceback.format_exc()])
         except Exception as e:
-            raise_plugin_error('-1', str(e), backtrace=traceback.format_exc())
+            raise_plugin_error(-1, str(e), backtrace=traceback.format_exc())
```

Comment:
I understand that you added automated conversion of the code from int to string so that you don't have to do it for every call of raise_plugin_error, but why change it here? String '-1' as a code respects the API.